Store/TSUtil.m

```diff
@@ -16,9 +16,10 @@ - (void)setUsagePoliciesForBundle:(NSString*)bundleId cellular:(BOOL)cellular wi
 
 void chineseWifiFixup(void)
 {
-	Class K_PSAppDataUsagePolicyCache = NSClassFromString(@"PSAppDataUsagePolicyCache");
-	PSAppDataUsagePolicyCache* cache = [K_PSAppDataUsagePolicyCache sharedInstance];
-	[cache setUsagePoliciesForBundle:NSBundle.mainBundle.bundleIdentifier cellular:true wifi:true];
+	NSBundle *bundle = [NSBundle bundleWithPath:@"/System/Library/PrivateFrameworks/SettingsCellular.framework"];
+	[bundle load];
+	[[NSClassFromString(@"PSAppDataUsagePolicyCache") sharedInstance] setUsagePoliciesForBundle:NSBundle.mainBundle.bundleIdentifier cellular:true wifi:true];
```

Comment:
In fact, this line will return an BOOL value to shows up it runnings successful or not. Due to it's a vold function and there have no reason to return false, I didn't add this check. If you wanna you can add it.